packages/env/lib/build-docker-compose-config.js

```diff
@@ -51,6 +51,10 @@ module.exports = function buildDockerComposeConfig( config ) {
 		...themeMounts,
 	];
 
+	// Set the default ports based on the config values.
+	const mainPort = `\${WP_ENV_PORT:-${ config.portNumber }}:80`;
```

Comment:
Let's name this `developmentPorts` to align with `developmentMounts` above.

Reply:
Done :)